tests/unit/test_cli.py

```diff
@@ -332,6 +334,27 @@ def test_migrate_credentials_azure(ws):
         ws.storage_credentials.list.assert_called()
 
 
+def test_migrate_credentials_aws(ws, mocker):
+    mocker.patch("shutil.which", return_value=True)
+    ws.config.is_azure = False
+    ws.config.is_aws = True
+    uc_trust_policy = mocker.patch(
+        "databricks.labs.ucx.assessment.aws.AWSResourcePermissions.update_uc_role_trust_policy"
+    )
+    with patch("databricks.labs.blueprint.tui.Prompts.confirm", return_value=True):
```

Comment:
use `MockPrompts` instead: https://github.com/databrickslabs/blueprint?tab=readme-ov-file#unit-testing-prompts

Reply:
we are not injecting Prompts() into cli at the moment, so will need to update that in blueprint to able to MockPrompts? separate PR for that?